pkgs/tools/admin/hedgedoc-cli/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, pkgs, stdenv, fetchFromGitHub }:
+let
+  pname = "hedgedoc-cli";
+  version = "1.0";
+in stdenv.mkDerivation {
+  inherit pname version;
+
+  buildInputs = with pkgs; [
```

Comment:
Instead of `with pkgs`, add those packages explicitly to the arguments list at the top. (and remove `pkgs` from the argumets)